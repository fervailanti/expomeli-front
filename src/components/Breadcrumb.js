import React from 'react'
import { images } from '../images'

const Breadcrumb = ({ categories = [] }) => (
  <div className="flex flex-wrap items-center mb-3 lg:mb-5">
    {categories.map((category, index) => [
      <p key={index} className="font-thin text-sm lg:text-base">
        {category}
      </p>,
      <img
        key={'img' + index}
        src={images.rightChevron}
        alt="arrow-icon"
        className="h-3 mx-2 last:hidden"
      />
    ])}
  </div>
)

export default Breadcrumb
