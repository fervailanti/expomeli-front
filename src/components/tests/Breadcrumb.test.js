import React from 'react'
import Breadcrumb from '../Breadcrumb'
import { render } from '@testing-library/react'

it('Breadcrumb component should render correctly', () => {
  const tree = render(<Breadcrumb categories={['Apple', 'iPhone']} />)
  expect(tree.container).toMatchSnapshot()
})
