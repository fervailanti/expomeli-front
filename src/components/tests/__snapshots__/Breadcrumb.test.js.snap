// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumb component should render correctly 1`] = `
<div>
  <div
    class="flex flex-wrap items-center mb-3 lg:mb-5"
  >
    <p
      class="font-thin text-sm lg:text-base"
    >
      Apple
    </p>
    <img
      alt="arrow-icon"
      class="h-3 mx-2 last:hidden"
      src="[object Object]"
    />
    <p
      class="font-thin text-sm lg:text-base"
    >
      iPhone
    </p>
    <img
      alt="arrow-icon"
      class="h-3 mx-2 last:hidden"
      src="[object Object]"
    />
  </div>
</div>
`;
